### DISCUSSION TITLE
Diapositivas/presentacion

### DISCUSSION THREAD

**Discussion by AnGys09** [INGCOM-UNRN-P1/cursada-2023#166] [General de la cátedra]
Buenas,  consulta las diapositivas de la clase (07)  de matrices/ struct,   están en algún lado, que no es el campus?? gracias.

**Comment by AnGys09** [comment]
Perdón que sea Molesta Pero seria genial, poder ver la presentación para ver de resolver unas cosas del tp!!! Gracias

**Reply by martinvilu** [reply]
Para nada, se nos pasó `¯\_(ツ)_/¯`

**Comment by LautaroTummino** [comment]
Sii, faltan las diapos de la ultima clase !!

Saludos !!

**Comment by martinvilu** [comment]
Presentación número 7
https://campusbimodal.unrn.edu.ar/mod/page/view.php?id=753986

El PDF esta subiendo a la carpeta en el campus.
